javascript-functions/main.js

```diff
@@ -10,8 +10,8 @@ function greet(name) {
   return name;
 }
 
-var myName = greet('Dre');
-console.log('Greeting:', 'Hey, ' + myName);
+var myName = greet('Hey, ' + 'Dre');
```

Comment:
This is still incorrect. As of right now, your `greet` function isn't doing anything except return the parameter's value right back out. The `hey, "` needs to be being added in the `greet` function itself, that is its purpose